src/test/java/org/apache/commons/lang3/StringUtilsTest.java

```diff
@@ -52,32 +52,6 @@
     static final String HARD_SPACE;
     static final String TRIMMABLE;
     static final String NON_TRIMMABLE;
-    static {
-        String ws = "";
-        String nws = "";
-        final String hs = String.valueOf(((char) 160));
-        String tr = "";
-        String ntr = "";
-        for (int i = 0; i < Character.MAX_VALUE; i++) {
-            if (Character.isWhitespace((char) i)) {
-                ws += String.valueOf((char) i);
-                if (i > 32) {
-                    ntr += String.valueOf((char) i);
-                }
-            } else if (i < 40) {
-                nws += String.valueOf((char) i);
-            }
-        }
-        for (int i = 0; i <= 32; i++) {
-            tr += String.valueOf((char) i);
-        }
-        WHITESPACE = ws;
-        NON_WHITESPACE = nws;
-        HARD_SPACE = hs;
-        TRIMMABLE = tr;
-        NON_TRIMMABLE = ntr;
-    }
-
```

Comment:
Please don't rearrange code if it is not necessary. It makes reviewing patches harder.